flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -380,23 +249,23 @@ const BAG_WEIGHT = 20;
      * Calculate %MAC ZWFCG of all stations
      */
 function getZfwcg() {
-    const currentPaxWeight = PAX_WEIGHT + BAG_WEIGHT;
+    const currentPaxWeight = PAX_WEIGHT;
 
-    const leMacZ = -5.39; // Value from Debug Weight
-    const macSize = 13.45; // Value from Debug Aircraft Sim Tunning
+    const leMacZ = -5.386; // Accurate to 3 decimals, replaces debug weight values
+    const macSize = 13.454; // Accurate to 3 decimals, replaces debug weight values
 
-    const emptyWeight = 101990 * 0.453592; // Value from flight_model.cfg to kgs
+    const emptyWeight = (SimVar.GetSimVarValue("EMPTY WEIGHT", "Number")) * 0.453592; // Value from flight_model.cfg to kgs
```

Comment:
I think changing this to 'const emptyWeight = SimVar.GetSimVarValue("Empty Weight", "kilograms")' should fix the double conversion issue. No need for any conversion as the kilogram value will always be read directly.

This same change needs to be made in line 293 below.

Reply:
Pushed the fix 👍🏻